### PR TITLE
Glassbr srs

### PR DESCRIPTION
**Pull request by vajihehm**
@JacquesCarette I have compiled and tested Drasil on "**glassbr_srs**" branch to update to give pascal unit to the LR,NFL.

### CODE REVIEW

File: code/drasil-example/Drasil/GlassBR/Unitals.hs [hunk 1]
```diff
@@ -170,14 +170,22 @@ sd_min     = mkQuantDef (unitary "sd_min"
...
+glassBRSymbols = [min_thick, sflawParamK, sflawParamM, demand, lRe, nonFactorL, load_dur,
   eqTNTWeight]
 
-min_thick, sflawParamK, sflawParamM, demand, sdx, sdy, sdz, load_dur,
+min_thick, sflawParamK, sflawParamM, demand, sdx, sdy, sdz, lRe, nonFactorL, load_dur,
   eqTNTWeight :: UnitaryChunk
 
 demand      = unitary "demand"      (nounPhraseSP "applied load (demand)")
   lQ pascal Rational --correct Space used?
+-----------------------------inserted by Vajiheh-----------------------------------------
```
Comment (JacquesCarette) [inline code/drasil-example/Drasil/GlassBR/Unitals.hs]: Please remove these comments - the 'diff' feature of git already makes that clear (i.e. what has been changed and by whom).

File: code/drasil-example/Drasil/GlassBR/Unitals.hs [hunk 2]
```diff
@@ -229,10 +237,11 @@ is_safeLR      = vc "is_safeLR"        (nounPhraseSP $ "variable that is assigne
 lDurFac       = vc'' (loadDurFactor) (Atomic "LDF") Real
 
 loadSF        = vc'' (lShareFac) (Atomic "LSF") Natural
+----------------------Should be moved to UnitryChunckSection-----------------------------------
```
Comment (JacquesCarette) [inline code/drasil-example/Drasil/GlassBR/Unitals.hs]: Please delete both the comment and the commented out code.

File: code/drasil-example/Drasil/GlassBR/Unitals.hs (hunk 1 above)
Comment (vajihehm) [reply code/drasil-example/Drasil/GlassBR/Unitals.hs]: @JacquesCarette it was addressed with 1cf95d5.

File: code/drasil-example/Drasil/GlassBR/Unitals.hs (hunk 2 above)
Comment (vajihehm) [reply code/drasil-example/Drasil/GlassBR/Unitals.hs]: @JacquesCarette it was addressed with 0bd1b19.

File: code/stable/glassbr/SRS/GlassBR_SRS.tex [hunk 3]
```diff
@@ -411,7 +415,7 @@ \subsubsection{Theoretical Models}
                    \\ \midrule \\
                    Description & \begin{symbDescription}
                                  \item{$is-safeLR$ is the variable that is assigned true when load resistance (capacity) is greater than load (demand) (Unitless)}
-                                 \item{$LR$ is the load resistance (Unitless)}
+                                 \item{$LR$ is the Load resistance (Pa)}
```
Comment (JacquesCarette) [inline code/stable/glassbr/SRS/GlassBR_SRS.tex]: load is now Load, that seems odd.

File: code/stable/glassbr/SRS/GlassBR_SRS.tex [hunk 4]
```diff
@@ -584,14 +588,14 @@ \subsubsection{Data Definitions}
...
-                 Units & Unitless
+                 Units & Pa
                          \\ \midrule \\
                          Equation & \begin{dmath}
                                     NFL=\frac{{\hat{q}_{tol}} E h^{4}}{\left(a b\right)^{2}}
                                     \end{dmath}
                                     \\ \midrule \\
                                     Description & \begin{symbDescription}
-                                                  \item{$NFL$ is the non-factored load (Unitless)}
+                                                  \item{$NFL$ is the Non-factored load (Pa)}
```
Comment (JacquesCarette) [inline code/stable/glassbr/SRS/GlassBR_SRS.tex]: Again, different capitalization going on here.

File: code/drasil-example/Drasil/GlassBR/Unitals.hs [hunk 5]
```diff
@@ -170,14 +170,21 @@ sd_min     = mkQuantDef (unitary "sd_min"
...
   eqTNTWeight]
 
-min_thick, sflawParamK, sflawParamM, demand, sdx, sdy, sdz, load_dur,
+min_thick, sflawParamK, sflawParamM, demand, sdx, sdy, sdz, lRe, nonFactorL, load_dur,
   eqTNTWeight :: UnitaryChunk
 
 demand      = unitary "demand"      (nounPhraseSP "applied load (demand)")
   lQ pascal Rational --correct Space used?
+  
+lRe      = unitary "lRe"      (nounPhraseSP "Load resistance")
```
Comment (JacquesCarette) [inline code/drasil-example/Drasil/GlassBR/Unitals.hs]: The problem might originate here -- our definitions all use lower-case only, and then combinators capitalize when needed.

File: code/drasil-example/Drasil/GlassBR/Unitals.hs [hunk 6]
```diff
@@ -170,14 +170,21 @@ sd_min     = mkQuantDef (unitary "sd_min"
...
+  (Atomic "LR") pascal Rational --correct Space used?
+
+nonFactorL      = unitary "nonFactorL"      (nounPhraseSP "Non-factored load")
```
Comment (JacquesCarette) [inline code/drasil-example/Drasil/GlassBR/Unitals.hs]: And here.

**Comment by JacquesCarette** [comment]
What is holding up making these changes?  They should take just a few minutes, and it's been 6 days.

**Comment by vajihehm** [comment]
> What is holding up making these changes? They should take just a few minutes, and it's been 6 days.

Sorry for the late fixing. I have some problems with the local build which caused I have not had successful build locally though.

**Comment by vajihehm** [comment]
@JacquesCarette I am writing to let you know as you mentioned in the last meeting we should open the different pull requests for the independent changes . However, after I have already pushed the changes (delete of the scope of the project) on the branch, I found that there can only be one pull request for a branch. So, I don't think there's a way to take it back but I can revert the change and then commit the revert. Thus, if I must do as such way please let me know. Otherwise, I will do this way for the future changes.

**Comment by JacquesCarette** [comment]
Please remove the commits 6e9416c and 7b7065e from this PR.  Commit de694d1 was correct and should have been the last one on here.
The correct protocol is to create a **new** branch for new work.
So now we are still in the state where I can't merge this in!

**Pull request merged**